File: 리트코드/0771-jewels-and-stones/0771-jewels-and-stones.java
class Solution {
    public int numJewelsInStones(String jewels, String stones) {
        Set<Character> freqs = new HashSet<>();

        for (char c : jewels.toCharArray()) {
            freqs.add(c);
        }

        int answer = 0;
        
        for (char c : stones.toCharArray()) {
            if (freqs.contains(c)) {
                answer++;
            }
        }
        return answer;
    }
}

File: 리트코드/0771-jewels-and-stones/0771-jewels-and-stones.kt
class Solution {
    fun numJewelsInStones(jewels: String, stones: String): Int {
    var answer = 0
    var freqs: MutableSet<Char> = HashSet()

    for (j in jewels.toCharArray()) {
        freqs.add(j)
    }
    for (s in stones.toCharArray()) {
        if(freqs.contains(s)) answer ++
    }
    return answer
}
}